### PR TITLE
New forum post shortcode for custom caption

### PR DESCRIPTION
**Pull request by rica-carv**
New forum post shortcode for custom caption

### CODE REVIEW

File: e107_plugins/forum/shortcodes/batch/post_shortcodes.php [hunk 1]
```diff
@@ -434,6 +434,29 @@ function sc_forum_post_breadcrumb()
...
+    if ($this->var['action'] == "nt")
+    {
+      $pre = LAN_FORUM_1001;
+			$name = strip_tags($tp->toHTML($this->var['forum_name'], false, 'no_hook, emotes_off'));
+			$url = e107::url('forum', 'forum', $this->var);
+      $post = LAN_FORUM_2005;
+    }
+    return $pre.($url?": <a {$title} href='".$url."'>{$name}</a> - ":$name).$post;
+	}
+
```
Comment (CaMer0n) [inline e107_plugins/forum/shortcodes/batch/post_shortcodes.php]: The only part I don't like is all that strip_tags() stuff.  Maybe you can simply use toHtml($this->var['forum_name'], false, 'TITLE');

Comment (rica-carv) [inline e107_plugins/forum/shortcodes/batch/post_shortcodes.php]: I was trying to get rid of the HTML on the variable, but anyway, i can live with it....

File: e107_plugins/forum/shortcodes/batch/post_shortcodes.php [hunk 2]
```diff
@@ -442,17 +442,17 @@ function sc_forum_post_captionname()
...
-      $post = LAN_FORUM_2006;
+      	$pre = LAN_FORUM_1003;
+	$name = $this->var['thread_name'];
+	$url = e107::url('forum', 'topic', $this->var);
+      	$post = LAN_FORUM_2006;
     }
     if ($this->var['action'] == "nt")
     {
-      $pre = LAN_FORUM_1001;
-			$name = strip_tags($tp->toHTML($this->var['forum_name'], false, 'no_hook, emotes_off'));
```
Comment (Moc) [inline e107_plugins/forum/shortcodes/batch/post_shortcodes.php]: Cameron meant using this 
```php
$tp->toHtml($this->var['forum_name'], false, 'TITLE');
```

So basically just remove the strip_tags() only 
Now, you have also removed the toHtml method. 
Same applies to the thread_name on line 445.

Comment (rica-carv) [inline e107_plugins/forum/shortcodes/batch/post_shortcodes.php]: Sorry, missed that. Corrected...

**Comment by CaMer0n** [comment]
Just change to `{FORUM_POST_CAPTION}` and it should be good to merge.

**Comment by rica-carv** [comment]
@CaMer0n Donne...

**Comment by CaMer0n** [comment]
Thank you!

**Pull request merged**